VimeoNetworking/VimeoNetworking/Request.swift

```diff
@@ -99,43 +99,20 @@ public struct Request<ModelType: MappableResponse>
         /// request url path (e.g. `/me`, `/videos/123456`)
     public let path: String
     
-        /// any parameters to include with the request, this is calculated from the raw parameters provided at request initialization plus any specified `page` or `itemsPerPage` values on the request itself
-    public var parameters: VimeoClient.RequestParameters
```

Comment:
Nope, no longer necessary. I'm removing pagination info from the Request class completely since not every (almost all) requests don't have anything to do with pagination (see my next comment).